solidity/contracts/converter/BancorConverterRegistry.sol

```diff
@@ -1,9 +1,11 @@
 pragma solidity 0.4.26;
+import '../utility/TokenHandler.sol';

```

Comment:
still need to rename TokenHandler to something more self describing like SafeTransfers

Reply:
@yudilevi : It implement functions `safeTransfer`, `safeTransferFrom` and `safeApprove`, so it doesn't make sense to name it `SafeTransfers`.

Reply:
it does cause all of these are related to safe transfers

Reply:
@yudilevi : I meant, it doesn't make sense IMO to do something like `SafeTransfers(token).safeTransfer`.

Reply:
we'll figure out the names but that generic name has to be replaced by something more self explanatory

Reply:
@yudilevi: I've opened a Trello task on it, so you can resolve this comment.